packages/clay-drop-down/src/__tests__/__snapshots__/DropDownWithDrilldown.tsx.snap

```diff
@@ -22,3 +22,26 @@ Document {
   },
 }
 `;
+
+exports[`ClayDropDownWithDrilldown renders dividers 1`] = `
+Document {
+  "location": Location {
+    "assign": [Function],
+    "hash": "",
+    "host": "localhost",
+    "hostname": "localhost",
+    "href": "http://localhost/",
+    "origin": "http://localhost",
+    "pathname": "/",
+    "port": "",
+    "protocol": "http:",
+    "reload": [Function],
+    "replace": [Function],
+    "search": "",
+    "toString": [Function],
+  },
+  Symbol(SameObject caches): Object {
+    "styleSheets": StyleSheetList {},
+  },
+}
```

Comment:
This is a little intriguing, I think it wasn’t meant to be rendered that way and I don’t know how I didn’t realize it before 🤷‍♂️. We may have to change everyone to render `document.body`

Reply:
I hadn't seen that either, let me have a look into it and get back to you.

Reply:
Yeah :see_no_evil: didn't even realize we were checking `document` and not `document.body`, updated in  [02be8af](https://github.com/liferay/clay/pull/4100/commits/02be8af2b2171891ad3ae0bfd922d9362213a9d5)